plugins/modules/system/runit.py

```diff
@@ -239,7 +217,7 @@ def main():
             name=dict(type='str', required=True),
             state=dict(type='str', choices=['killed', 'once', 'reloaded', 'restarted', 'started', 'stopped']),
             enabled=dict(type='bool'),
-            dist=dict(type='str', default='runit'),  # @FIXME unused param?
+            dist=dict(type='str', removed_in_version='4.0.0', removed_from_collection='community.general'),
```

Comment:
Can you put this into a separate PR? This is the only thing we cannot backport to stable-1.